apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts

```diff
@@ -165,6 +165,12 @@ export class SelectMultipleFieldsComponent implements OnInit, OnDestroy, OnChang
     public removeOption(index: number): void {
         if (index >= 0) {
             this.chipList.splice(index, 1);
+            // Close the autocomplete dropdown if it's open
+            setTimeout(() => {
+                if (this.trigger && this.trigger.panelOpen) {
```

Comment:
setTimeout k time agr this.trigger milta tha to set timeout ko apn if block andr dal skte h

Reply:
nh milta h